cadc-sia/src/main/java/ca/nrc/cadc/sia2/impl/ServiceAvailability.java

```diff
@@ -187,18 +195,20 @@ public static String getTapURI() {
     }
 
     private static String getTapURIProperty() throws IOException {
-        URL url = SiaRunner.class.getResource(CONFIG_FILE_NAME);
-        if (url == null) {
-            url = SiaRunner.class.getResource("/" + CONFIG_FILE_NAME);
-        }
+        // Try to get from a file on disk first.
+        final PropertiesReader propertiesReader = new PropertiesReader(CONFIG_FILE_NAME);
 
-        if (url == null) {
-            final PropertiesReader propertiesReader = new PropertiesReader(CONFIG_FILE_NAME);
+        if (propertiesReader.canRead()) {
             final MultiValuedProperties multiValuedProperties = propertiesReader.getAllProperties();
```

Comment:
PropertiesReader has a getFirstValue(key) method that would allow you to make these checks a little cleaner.